app/api/v1/endpoints/roles.py

```
@@ -32,14 +72,125 @@ async def list_roles(
+    if body.description is not None:
+        role.description = body.description
+    if body.is_active is not None:
+        role.is_active = body.is_active
+    updated = await role_repo.update(role)
+    return RoleResponse.model_validate(updated)
+
+
+@router.delete("/{role_id}", status_code=204)
+@limit_writes
+async def delete_role(
+    request: Request,
```

Ruff (0.15.0)

[warning] Unused function argument: `request`

(ARG001)